mecanum_drive_controller/test/mecanum_drive_controller_params.yaml

```diff
@@ -0,0 +1,27 @@
+test_mecanum_drive_controller:
+  ros__parameters:
+
+    reference_timeout: 0.1
+
+    command_joint_names: ["front_left_wheel_joint", "back_left_wheel_joint", "back_right_wheel_joint", "front_right_wheel_joint"]
+
+    reference_names: ["linear_x", "linear_y", "angular_z"]
```

Comment:
We should hard-code those into the controller. No need to define the reference names here.

Reply:
i asked you already a question on this, probably we missed it, my question is, in general this is applicable should i include this in templates ?

Reply:
In general, this should be in the templates. In this particular case, it doesn't make sense to have this configurable.

Reply:
Can we delete this?

```suggestion
```

Reply:
sorry i alread did but not yet pushed, i did not ask for review from my side yet.